app/services/return-requirements/check/returns-requirements.service.js

```diff
@@ -16,12 +17,13 @@ const ReturnRequirementsPresenter = require('../../../presenters/return-requirem
  * @returns {Promise<Object>} page data needed by the view template
  */
 async function go (session) {
-  const { requirements, journey } = session.data
+  const { licence, data: { requirements, journey } } = session
```

Comment:
Sorry, I didn't spot on the previous PR that we were calling `session.data`.

Both `requirements` and `licence` should exist at the top level of session. So, you should be able to just do

```suggestion
  const { licence, journey, requirements } = session
```